website/content/docs/agent/options.mdx

```diff
@@ -242,8 +242,8 @@ The options below are all specified on the command-line.
   If it is provided after Consul has been initialized with an encryption key, then
   the provided key is ignored and a warning will be displayed.
 
-- `-grpc-port` ((#\_grpc_port)) - the gRPC API port to listen on. Default
```

Comment:
Since our docs aren't versioned yet, I think we need to keep the two old grpc lines in here, but mark them as deprecated and link them to the new names.

Reply:
Unfortunate, but makes sense

Reply:
Usually when we deprecate stuff like this we also indicate which version of consul it was deprecated in so that folks using older versions can still figure out what's going on for their version.

Reply:
Ah, sorry I missed this comment. I wasn't sure which version this would land in yet. I guess it will be 1.11, so I can add that.

Reply:
https://github.com/hashicorp/consul/pull/10618